src/main/java/se/bjurr/gitchangelog/internal/git/GitRepo.java

```diff
@@ -585,34 +615,6 @@ private boolean shouldPrioritizeNewWork(
     return shouldPrioritizeNewWork(existingTagName, newTagName);
   }
 
-  static boolean shouldPrioritizeNewWork(final String existingTagName, final String newTagName) {
```

Comment:
There are several changes in this PR where you move stuff around. Can you revert those changes? It makes the diff hard to read.

Reply:
I am having some problems with my google code formatter plugin. I will revert.